opencti-platform/opencti-front/src/private/components/data/connectors/ConnectorWorks.tsx

```diff
@@ -0,0 +1,388 @@
+                    <Tooltip title={work.name}>
+                      <Typography>
+                        {truncate(work.name, 80)}
+                      </Typography>
+                    </Tooltip>
+                  </Grid>
+                  <Grid item={true} xs={6}>
+                    <Typography variant="h3" gutterBottom={true}>
+                      {t_i18n('Status')}
+                    </Typography>
+                    <TaskStatus status={work.status} label={t_i18n(work.status)} />
+                  </Grid>
+                  <Grid item={true} xs={6}>
+                    <Typography
+                      variant="h3"
+                      gutterBottom={true}
+                      style={{ marginTop: 20 }}
+                    >
+                      {t_i18n('Work start time')}
+                    </Typography>
+                    {nsdt(work.received_time)}
+                  </Grid>
+                  <Grid item={true} xs={6}>
+                    <Typography
+                      variant="h3"
+                      gutterBottom={true}
+                      style={{ marginTop: 20 }}
+                    >
+                      {t_i18n('Work end time')}
+                    </Typography>
+                    {work.completed_time ? nsdt(work.completed_time) : '-'}
+                  </Grid>
+                </Grid>
+              </Grid>
+              <Grid item={true} xs={5}>
+                <Grid container={true} spacing={3}>
+                  <Grid item={true} xs={6}>
+                    <Typography variant="h3" gutterBottom={true}>
+                      {t_i18n('Operations completed')}
+                    </Typography>
+                    <span className={classes.number}>
+                      {work.status === 'wait'
+                        ? '-'
+                        : tracking?.import_processed_number ?? '-'}
+                    </span>
+                  </Grid>
+                  <Grid item={true} xs={6}>
+                    <Typography variant="h3" gutterBottom={true}>
+                      {t_i18n('Total number of operations')}
+                    </Typography>
+                    <span className={classes.number}>
+                      {work.status === 'wait'
+                        ? '-'
+                        : tracking?.import_expected_number ?? '-'}
+                    </span>
+                  </Grid>
+                  <Grid item={true} xs={12}>
+                    <Typography variant="h3" gutterBottom={true}>
+                      {t_i18n('Progress')}
+                    </Typography>
+                    <LinearProgress
+                      classes={{ root: classes.progress }}
+                      variant="determinate"
+                      value={
+                        // eslint-disable-next-line no-nested-ternary
+                        tracking && tracking.import_expected_number !== null && tracking.import_processed_number !== null
```

Comment:
this issue here is to check for the existence of the field, but as it's a number we shall not fail the test on 0 value.

Maybe use ` tracking && Number.isFinite(tracking.import_expected_number) && Number.isFinite(tracking.import_processed_number) && `

Reply:
in fact using `!!tracking.import_expected_number && !!tracking.import_processed_number` might be sufficient in this case, because the fallback is `: 0` anyway, wichi is the result of the computation in any case.

Reply:
I tried with `!!`, and removed the 2nd check in the ternary operator since it was already solved by the first. It seems to work correctly with `null` and `undefined`, returning 0 in all cases.